subsys/bluetooth/controller/Kconfig

```diff
@@ -456,6 +456,7 @@ config BT_CTLR_DATA_LENGTH
 config BT_CTLR_DATA_LENGTH_MAX
 	int "Maximum data length supported"
 	depends on BT_CTLR_DATA_LENGTH
+	default 37 if BT_MESH
```

Comment:
Hm... Isn't this valid anymore? https://github.com/zephyrproject-rtos/zephyr/pull/54681/files#r1111895307

Reply:
True, ok, lets remove this for now and solve in NCSDK-18316.